modules/ui/src/main/scala/lucuma/ui/primereact/EnumDropdownView.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import crystal.react.View
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import react.common.*
+import react.primereact.Dropdown
+import react.primereact.SelectItem
+import reactST.primereact.components.{Dropdown => CDropdown}
+
+import scalajs.js
+import scalajs.js.JSConverters.*
+
+object EnumDropdownView:
```

Comment:
Since we're going to hide the builders, I don't think there is a good reason to keep the `non-Form` versions of the `*DropDownView`s. The biggest reason for their existence was that they exposed the builder.